### DISCUSSION TITLE
Sentinel-2 reprocessed data availability

### DISCUSSION THREAD

**Discussion by seleneledain** [microsoft/PlanetaryComputer#340] [Data request]
I am wondering whether the Sentinel-2 L2A catalogue will be updated to include re-processed historical data according to the latest processing baseline. This is currently being done by Copernicus (https://sentinels.copernicus.eu/web/sentinel/technical-guides/sentinel-2-msi/copernicus-sentinel-2-collection-1-availability-status) and should be completed for Q2 2024. This would include for example geometric refinement,

Do you plan to update the dataset or leave the products with their older processing baselines?

**Comment by TomAugspurger** [comment]
Thanks for raising this. We've talked a bit about what to do, but don't have a concrete plan yet.

I'd be curious to hear your (and others') thoughts on how best to approach this. If you had to pick just one version of sentinel-2, would Collection-1 be it?

**Reply by seleneledain** [reply]
All right thanks! This is very user dependent, but it could be convenient to have all data with the same processing baseline to have consistency across multi-year datasets. This facilitates any downstream processing since the same pipeline can be applied everywhere. So this could be for example Collection 1 with processing baseline 05.xx

**Comment by gary-downforce** [comment]
Just to chime in on this, I'd love to have the full Collection-1 data available via Planetary Computer. That way there's no need for harmonising the data when comparing between different years, the accuracy of geolocation is the same across the dataset due to consistent DEM etc.

It also makes sense because ESA are currently deleting the old data now that they've finished the reprocessing.

**Reply by 777arc** [reply]
Thanks for chiming in, mind creating a new Issue in GitHub requesting this specific data so we can easily track it.  Thanks!
